dvc/command/base.py

```diff
@@ -44,7 +44,8 @@ def __init__(self, args):
         self.repo = Repo()
         self.config = self.repo.config
         self.args = args
-        updater = Updater(self.repo.dvc_dir)
+        nfs_lock = self.config.config["core"].get("nfs_lock", False)
```

Comment:
This is ridiculous, need to revisit our config to make it more usable. 😕

Reply:
@efiop , want to create an issue about it?

Reply:
@mroutis Sure, I'll create one later 🙂

Reply:
There is a helper already or am I wrong?
```python
self.config.get('core', 'nfs_lock')  # add False as default to schema
```

Reply:
@Suor Good point! In this case config is always "verified" (passed through schema check that sets some defaults) so we are safe using config.get here. Thanks!

Reply:
@Suor Actually we can't as we use unverified configs there from different config levels. Not the time to fix that in this PR. Just using a straightforward approach for now, same as in other code. Will be properly fixed in the following config refactorings.